test/models/name_test.rb

```diff
@@ -96,7 +96,7 @@ def assert_name_match_various_authors(pattern, string, first_match)
   end
 
   def assert_name_match(pattern, string, first, second = "")
-    match = pattern.match(string)
+    match = pattern.to_s.match?(string)
```

Comment:
```suggestion
    match = pattern.match(string)
```
The subsequent lines require the `match` local variable to be a MatchData object.

Reply:
sorry i was on autopilot. thanks for catching this.